### PR TITLE
Better progress reporting for pending/scheduled plans

### PR DESCRIPTION
**Pull request by iNecas**
They should be always 0.

Also, I hit an issue with a state, where the plan was in scheduled state, but
the scheduler record was not there: I updated the dynflow console to handle
this error better.

### CODE REVIEW

**Comment by adamruzicka** [comment]
Looks good to me.

Do you know how to reproduce the issue with missing schedule record?

**Comment by iNecas** [comment]
@adamruzicka after writting tests I've realised we should really handle the errors in the serializer better, which lead me to some bigger changes, including the serializer interface, so that we can perform the serialization while catching the errors, and then access the serialized args when creating the schedule object. I've also hit some other issues, that I've fixed as well

File: test/future_execution_test.rb [hunk 1]
```diff
@@ -94,13 +105,13 @@ module FutureExecutionTest
       end
 
       describe 'serializers' do
-        let(:save_and_load) do
```
Comment (adamruzicka) [inline test/future_execution_test.rb]: Are we sure that the `serialized_args` won't be changed by saving/loading it to/from a database?

Comment (iNecas) [inline test/future_execution_test.rb]: Ah, I first thouht this was a dead-code (missed the reference to it in the oirginal test), I've put it back

**Comment by adamruzicka** [comment]
Looks good to me

**Comment by iNecas** [comment]
@adamruzicka thanks for the review and coments!

**Pull request merged**